packages/client/src/pages/projects/NewProject.tsx

```diff
@@ -59,13 +62,22 @@ export const NewProject: React.FC = () => {
       setError(false);
     }
   };
-  const handleSubmit = (event: { preventDefault: () => void }) => {
+  const handleSubmit = async (event: { preventDefault: () => void }) => {
     if (error) {
       setError(true);
       event.preventDefault();
       return;
     } else {
-      //submit logic
+      const result = await apolloClient.mutate({
+        mutation: CreateProjectDocument,
+        variables: { project: data }
+      });
+
+      if (result.errors || !result.data) {
```

Comment:
Before running the mutation, you should check to see if the project name is unique. Right now the user wouldn't know. You can checkout `NewStudyJsonForm.component.tsx` to see how the error message can be added into the form directly